watchdog/schemas/server.yaml

```diff
@@ -0,0 +1,42 @@
+%YAML 1.1
+---
+"$schema": "http://json-schema.org/draft-06/schema#"
+title: Watchdog daemon role
+description: |
+  Watchdog daemon, server role.
+type: object
+additionalProperties: false
+
+required:
+- enabled
+
+properties:
+  enabled:
+    description: |
+      Enables Watchdog daemon service.
+    type: boolean
+  timeout:
+    description: |
+      Seting "watchdog-timer" config parameter placed in /etc/watchdog.conf
+    type: number
+    minimum: 0
+  kernel:
+    description: |
+      Parameters for the kernel module which are used for loading the kernel module
+    type: object
+    parameters:
+      parameter:
```

Comment:
```
parameters:
  parameter:
```
this is probably wrong?

Reply:
Not sure, but it should define the following structure:
```
    watchdog:
      server:
        kernel:
          parameter:
            soft_panic: 1
```

It means that `paratemetr` is one of possible "kernel" `parameters` - then it looks fine to me...